reflex/state.py

```diff
@@ -213,6 +213,18 @@ class BaseState(Base, ABC, extra=pydantic.Extra.allow):
     # The router data for the current page
     router: RouterData = RouterData()
 
+    # Custom exception hanlder
```

Comment:
This and the following lines will not be needed anymore if you move the exception_handlers to `rx.App` as suggested above